### PR TITLE
[Improve] Replace All Instances of "StreamPark" with "Apache StreamPark" in Official Documentation

### PR DESCRIPTION
**Pull request by Casonhqc**
close: [#3492](https://github.com/apache/incubator-streampark/issues/3492)

### CODE REVIEW

**Comment by caicancai** [comment]
@tisonkun Hello, should we replace with Apache StreamPark™ or Apache StreamPark

**Comment by caicancai** [comment]
@Casonhqc Thank you for your contribution, goo job

**Comment by VampireAchao** [comment]
Please add "™", such as "Apache StreamPark™"

**Comment by caicancai** [comment]
@Casonhqc Hello, I'm sorry that I didn't explain it clearly in the issue. Could you please replace Apache StreamPark with Apache StreamPark™? You can submit another PR to complete this matter.

File: blog/1-flink-framework-streampark.md [hunk 1]
```diff
@@ -56,18 +56,18 @@ However, because object storage requires the entire object to be rewritten for r
...
-This is an era of open-source uprising. Naturally, we turned our attention to the open-source realm: among numerous open-source projects, after comparing various projects, we found that both Zeppelin and StreamPark provide substantial support for Flink and both claim to support Flink on K8s. Eventually, both were shortlisted for our selection. Here's a brief comparison of their support for K8s (if there have been updates since, please kindly correct).
+This is an era of open-source uprising. Naturally, we turned our attention to the open-source realm: among numerous open-source projects, after comparing various projects, we found that both Zeppelin and Apache StreamPark™ provide substantial support for Flink and both claim to support Flink on K8s. Eventually, both were shortlisted for our selection. Here's a brief comparison of their support for K8s (if there have been updates since, please kindly correct).
 
 <table>
     <thead>
         <tr>
             <td>Feature</td>
             <td>Zeppelin</td>
-            <td>StreamPark</td>
+            <td>Apache StreamPark™</td>
```
Comment (caicancai) [inline blog/1-flink-framework-streampark.md:70]: No changes here, keep it as is

File: blog/1-flink-framework-streampark.md [hunk 2]
```diff
@@ -123,15 +123,15 @@ This is an era of open-source uprising. Naturally, we turned our attention to th
 
 <br/>
 
-During our research process, we communicated with the main developers of both tools multiple times. After our repeated studies and assessments, we eventually decided to adopt StreamPark as our primary Flink development tool for now.
+During our research process, we communicated with the main developers of both tools multiple times. After our repeated studies and assessments, we eventually decided to adopt Apache StreamPark™ as our primary Flink development tool for now.
 
 <video src="http://assets.streamxhub.com/streamx-video.mp4" controls="controls" width="100%" height="100%"></video>
 
-<center style={{"color": "gray"}}>(StreamPark's official splash screen)</center>
+<center style={{"color": "gray"}}>(Apache StreamPark™'s official splash screen)</center>
```
Comment (caicancai) [inline blog/1-flink-framework-streampark.md:130]: same

**Review by caicancai — CHANGES_REQUESTED** [review]

**Comment by caicancai** [comment]
@Casonhqc Hello, in the issue, you may only need to change it in the official blog, and other places will not be modified for the time being.

**Comment by caicancai** [comment]
You may submit a new PR to complete this issue

**Comment by tisonkun** [comment]
The original guide is https://www.apache.org/foundation/marks/guide#howoften.

The full form of an Apache project is "Apache Foo™" o "Apache Foo®"(if the trademark is registered). It should be used for the first and most prominent mentions, at least including:

* Titles or subtitles, including web page title or description metadata.
* The first and most prominent header elements within any major document section.
* The first and most prominent callout, sidebar, or other types of highlighted blocks of content displayed to the user.
* The first and most prominent uses in running or body text within the document.

Other mentions can use short form depending on the context and writing style.

Of course, using "Apache StreamPark" always and "Apache StreamPark™" for the first and most prominent mentions are correct, although it may sometimes become cumbersome.

The important part here is to make the decision based on the understanding of the policy, instead of taking others' words as a magic to pass the check.

**Comment by caicancai** [comment]
@Casonhqc You can follow this rule for your next PR, thank you
closing

> The original guide is https://www.apache.org/foundation/marks/guide#howoften.
> 
> The full form of an Apache project is "Apache Foo™" o "Apache Foo®"(if the trademark is registered). It should be used for the first and most prominent mentions, at least including:
> 
> * Titles or subtitles, including web page title or description metadata.
> * The first and most prominent header elements within any major document section.
> * The first and most prominent callout, sidebar, or other types of highlighted blocks of content displayed to the user.
> * The first and most prominent uses in running or body text within the document.
> 
> Other mentions can use short form depending on the context and writing style.
> 
> Of course, using "Apache StreamPark" always and "Apache StreamPark™" for the first and most prominent mentions are correct, although it may sometimes become cumbersome.
> 
> The important part here is to make the decision based on the understanding of the policy, instead of taking others' words as a magic to pass the check.

**Pull request closed without merging**